Makefile

```diff
@@ -0,0 +1,45 @@
+version     ?= latest
+log          = 'error'
+devimg       = sysdigclidev
+GOPATH      ?= $(HOME)/go
+packagename  = github.com/NeowayLabs/sysdigcli
+workdir      = /go/src/$(packagename)
+rundev       = docker run --net=host -e SEVERINO_LOGGER=$(log) --rm -v `pwd`:$(workdir) --workdir $(workdir) $(devimg)
+runbuild     = --rm -v `pwd`:$(workdir) -w $(workdir) $(devimg)
+gitversion   = $(version)
+
+all: check-integration analyze
+
+guard-%:
+	@ if [ "${${*}}" = "" ]; then \
+		echo "Variable '$*' not set"; \
+		exit 1; \
+	fi
+
+release: guard-version publish
+	git tag -a $(version) -m "Generated release "$(version)
+	git push origin $(version)
+
+imagedev:
+	docker build -t $(devimg) -f ./hack/Dockerfile.dev .
+
+vendor: imagedev
+	$(rundev) ./hack/vendor.sh
+	sudo chown -Rf $(USER):$(USER) ./vendor
+	sudo chown -Rf $(USER):$(USER) ./Gopkg.toml
+	sudo chown -Rf $(USER):$(USER) ./Gopkg.lock
+
+analyze: imagedev
```

Comment:
Could you add travis ci here in the GitHub, just create an account in the travis ci and enable the project :)